src/tests/suite/daos_epoch_recovery.c

```diff
@@ -141,23 +141,24 @@ epoch_recovery(test_arg_t *arg, enum epoch_recovery_op op)
 
 	MPI_Barrier(MPI_COMM_WORLD);
 
-	if (op == POOL_EVICT) {
-		if (arg->myrank == 0) {
-			print_message("evicting pool connections\n");
-			rc = daos_pool_evict(arg->pool.pool_uuid, arg->group,
-					     NULL /* svc */, NULL /* ev */);
-			assert_int_equal(rc, 0);
-		}
-		MPI_Barrier(MPI_COMM_WORLD);
-	}
+	//if (op == POOL_EVICT) {
+	//	if (arg->myrank == 0) {
+	//		print_message("evicting pool connections\n");
+	//		rc = daos_pool_evict(arg->pool.pool_uuid, arg->group,
+	//				     NULL /* svc */, NULL /* ev */);
+	//		assert_int_equal(rc, 0);
```

Comment:
(style)  Block comments use * on subsequent lines
(style)  do not use C99 // comments